src/app/components/core/backup-restore/backup-restore.component.html

```diff
@@ -72,7 +72,7 @@
             <tbody>
               <tr *ngFor="let backup of backupData">
                 <td class="align-content">
-                  <small>{{ backup.date | dateparser:"YYYY-MM-DD hh:mm:ss"}}</small>
+                  <small>{{ backup.date | dateparser:"YYYY-MM-DD HH:mm:ss"}}</small>
```

Comment:
were we showing incorrectly by transforming to 12hour format?

Reply:
yes, it was in 12hour format